kore/app/exec/Main.hs

```diff
@@ -374,18 +374,26 @@ mainWithOptions
                                             searchPattern
                                             searchConfig
                                     return (ExitSuccess, pat)
-                        Just specIndexedModule ->
-                            either
-                                (\pat -> (ExitFailure 1, pat))
-                                (\_ ->
-                                        ( ExitSuccess
-                                        , mkTop $ mkSortVariable "R"
+                        Just (specIndexedModule, bmc) ->
```

Comment:
I think this section could be clearer:
```.hs
                        Just (specIndexedModule, bmc)
                          | bmc -> do
                            _ <- boundedModelCheck
                                stepLimit
                                indexedModule
                                specIndexedModule
                            return success
                          | otherwise ->
                            either failure (const success)
                            <$> prove
                                stepLimit
                                indexedModule
                                specIndexedModule
                          where
                            failure pat = (ExitFailure 1, pat)
                            success = (ExitSuccess, mkTop $ mkSortVariable "R")
```

Reply:
done